support/routes/v2/isobot/ops/botService.ts

```diff
@@ -91,6 +170,276 @@ class BotService {
 
     await this.whitelistService.addWhitelist(emails)
   }
+
+  getValidatedDomain(payload: SlackPayload) {
+    const { text: domain } = payload
+    const DOMAIN_NAME_REGEX = /^[a-zA-Z0-9][a-zA-Z0-9-]{0,61}[a-zA-Z0-9](?:\.[a-zA-Z]{2,})+$/
```

Comment:
damn where did 61 come from

Reply:
Domain names have a character limit of 63 characters per part, so the 61 comes from the 63 minus the first and last character.